Source/IO.H

```diff
@@ -1,4 +1,25 @@
 #ifndef IO_H
 #define IO_H
+
+#include <AMReX_EB2.H>
+#include <AMReX_PlotFileUtil.H>
+#ifdef PELEC_USE_HDF5
+#include <AMReX_PlotFileUtilHDF5.H>
+#endif
+
 extern std::string inputs_name;
+
+class PeleCAmr : public amrex::Amr
```

Comment:
I would be fine moving this to `PeleCAmr.cpp/H` file. That way IO.cpp is shorter and we don't include IO.H in the `main.cpp`. Might help in recompile times.

Reply:
Yeah I can do that.